### ISSUE TITLE
Remove numerics from the file HmtlFileparser

### ISSUE THREAD

**Issue by vkuppu** [vkuppu/NLP-Natural-Language-Processsing-Implementation#7]
We may not require numeric for text analysis...

if needed we can revert back..

**Comment by Deakhadevi** [comment]
I don't think so. Suppose we analyse time means we require number . For time being we can remove the numeric . Further suppose we required means we can include in it.

**Comment by vkuppu** [comment]
I do Agree ..
Right ,at this point iam looking to simply our process.
As need be , we can keep updating features..
